### DISCUSSION TITLE
求问embed消息咋发送的呢

### DISCUSSION THREAD

**Discussion by wutitaxue** [simple-robot/Discussions#178] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

QQ频道

### 疑问描述

![image](https://github.com/simple-robot/Discussions/assets/50535063/7b72859a-709f-4b91-812d-d8c3f98ce23b)
求问发送这个要怎么实现啊

**Comment by ForteScarlet** [comment]
在底层API [`MessageSendApi`](https://docs.simbot.forte.love/components/qq-guild/simbot-component-qq-guild-api/love.forte.simbot.qguild.api.message/-message-send-api/index.html) 中可以使用属性 [`Body.embed`](https://docs.simbot.forte.love/components/qq-guild/simbot-component-qq-guild-api/love.forte.simbot.qguild.api.message/-message-send-api/-body/embed.html?query=val%20embed:%20Message.Embed?) 来提供 embed 消息的内容。不过这需要直接操作 `MessageSendApi`。

在核心库中，对 `embed` 提供直接支持的消息类型被遗漏了。可以暂时直接操作 `MessageSendApi` 来进行操作，后续会更新对 `embed` 进行直接支持的消息类型。

可以追踪 https://github.com/simple-robot/simbot-component-qq-guild/issues/107 来了解进度、提供帮助或评论。

**Reply by ForteScarlet** [reply, marked as answer]
https://github.com/simple-robot/simbot-component-qq-guild/pull/108

**Comment by wutitaxue** [comment]
感谢，可以发啦
